coops-ui/src/views/AnnualReport.vue

```diff
@@ -540,16 +537,26 @@ export default {
       this.setValidateFlag()
     },
 
+    directorsChange: function (val) {
+      // when directors change, update filing data
+      // console.log('AnnualReport, directorsChange =', val)
+      if (val) {
+        this.toggleFiling('add', 'OTCDR')
+      } else {
+        this.toggleFiling('remove', 'OTCDR')
+      }
+    },
+
     directorFormValid (val) {
       this.setValidateFlag()
     },
 
     validated (val) {
-      console.log('AnnualReport, validated =', val)
+      // console.log('AnnualReport, validated =', val)
```

Comment:
Can we delete the watchers that are now doing nothing? Not sure they ever should have been committed.

Reply:
Yep, done. Refresh the page to see the new commits.